build/commands/scripts/sync.js

```diff
@@ -41,40 +49,189 @@ const maybeInstallDepotTools = (options = config.defaultOptions) => {
       'is-googler': false,
       'version': 3,
       'countdown': 10,
-      'opt-in': false
+      'opt-in': false,
     };
     fs.writeFileSync(ninjaLogCfgPath, JSON.stringify(ninjaLogCfgConfig))
   }
 }
 
+function buildDefaultGClientConfig() {
```

Comment:
We typically use arrow functions, but not sure if there are guidelines for one vs the other. @petemill ?